hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTargetFilterQueryManagement.java

```diff
@@ -248,8 +249,10 @@ public TargetFilterQuery updateAutoAssignDS(final AutoAssignDistributionSetUpdat
             // auto-assign distribution set when creating a target filter query
             assertMaxTargetsQuota(targetFilterQuery.getQuery());
             final JpaDistributionSet ds = findDistributionSetAndThrowExceptionIfNotFound(update.getDsId());
+            final String initiatedBy = TenantAwareHolder.getInstance().getTenantAware().getCurrentUsername();
```

Comment:
I would inject the TenantAware directly via constructor injection here. Additionally I would inline the call into #setAutoAssignInitiatedBy